### PR TITLE
Add Windows build instructions to Readme

### PR DESCRIPTION
**Pull request by ghost**
Signed-off-by: Oxygemo <alej0hio2007@gmail.com>

### CODE REVIEW

**Comment by Peter0x44** [comment]
IIRC, I needed to select "thread model" as win32 in the installer, perhaps add a note to do that
I don't think it works with winpthreads
There also isn't any reason visual studio shouldn't work either, I just haven't tried it.

**Comment by ghost** [comment]
I came across an error when I used Visual Studio to compile it. I know the solution, which is to edit CMakeLists.txt, but @Devnol told me to just use MinGW and I agree with that.
> IIRC, I needed to select "thread model" as win32 in the installer, perhaps add a note to do that

In the MinGW installer? I never selected that and everything works here.

**Comment by Peter0x44** [comment]
> I came across an error when I used Visual Studio to compile it

Can you show what this error is? I'd like to fix it if possible.

**Comment by ghost** [comment]
image

**Comment by ghost** [comment]
https://stackoverflow.com/questions/2274006/cmake-invalid-numeric-argument-wextra#2274040

There is this on StackOverflow

**Comment by Peter0x44** [comment]
try commenting out line 15 of CMakeLists.txt
Those two lines turn on all the possible warnings, I guess msbuild has a different set of command line switches to do that
It should still build fine without it

**Comment by ghost** [comment]
That works. Should I add a section for MSVC?

**Comment by Peter0x44** [comment]
Sure, if you wish to do that

**Comment by ghost** [comment]
image
oh wait lol

**Comment by ghost** [comment]
Yeah, I don't think I'll add a section for that considering above

**Comment by Peter0x44** [comment]
This is actually a bug visual studio caught that gcc and clang didn't (probably using one of the sanitizers would have worked but I didn't bother)
I think I fixed it in 0135cc59c1471a447c78845dc5656b37c0ce0e3f

**Comment by ghost** [comment]
Alright, but is there anything wrong or something I should change with the PR?

File: README.md [hunk 1]
```diff
@@ -44,7 +44,23 @@ You should use PowerShell for these commands as it's just a bit better and that'
...
-7. Run GeometrySlash.exe! You have just compiled GeometrySlash for Windows.
+7. Run GeometrySlash.exe! You have just compiled GeometrySlash for Windows using MinGW.
 
+## Building (Windows, MSVC)
+
+We still recommend you to use CMake to compile GeometrySlash, you can get it from [here](https://cmake.org/download/). It's also possible to get it on Scoop, Winget or Chocolatey.
+
+Still, try to use PowerShell because it's much better than Command Prompt.
+
+You need to have Visual Studio 2019 (2017 miiiiiiight work? I'm not sure.) with C++ Development selected in the installer. It doesn't matter if you're using Community, Enterprise bla bla bla.
```
Comment (Peter0x44) [inline README.md]: Just say something like "versions earlier than visual studio 2019 are untested"

> It doesn't matter if you're using Community, Enterprise bla bla bla.

I don't think this honestly needs to be said. If you think it's necessary, just put something like "There is no functional difference between visual studio enterprise and community"

File: README.md [hunk 2]
```diff
@@ -44,7 +44,23 @@ You should use PowerShell for these commands as it's just a bit better and that'
...
+
+1. Clone the repository using Git. Execute this in a terminal: `git clone https://github.com/Peter0x44/GeometrySlash`
+2. Cd into GeometrySlash. `cd GeometrySlash`
+3. Install the Raylib git submodule by running this: `git submodule update --init --recursive`
+4. Make a build folder. Your builds will go here. `mkdir build; cd build`
+5. If line 15 of CMakeLists.txt isn't commented out, comment it out. Microsoft's compiler won't compile the project with those flags enabled.
```
Comment (Peter0x44) [inline README.md]: I will fix the CMakeLists.txt so commenting out that line is not necessary
Also, just so you know, `git clone --recursive` will also clone all submodules, so you can save a step

**Comment by Peter0x44** [comment]
As a more general thing, can you split it in two sections, titled "Building on WIndows with Visual Studio 2019" and "Building on Windows with MinGW-w64"?

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md]: Done.

**Comment by Peter0x44** [comment]
Thanks, lgtm

**Pull request merged**